### DISCUSSION TITLE
Save uploader model

### DISCUSSION THREAD

**Discussion by diego-lipinski-de-castro** [spatie/laravel-medialibrary#3340] [Q&A]
What would be the ideal way to add a morph column that saves the uploader model? What should I change in the Media model for that to work?

**Comment by parallels999** [comment]
what do you mean with "uploader model"?

https://github.com/spatie/laravel-medialibrary/blob/293a066890e10c951ff29bc76bfa026a60034edc/database/migrations/create_media_table.php.stub#L14

**Comment by diego-lipinski-de-castro** [comment]
I mean the user who is uploading the media, the owner

**Comment by parallels999** [comment]
Everything is customizable, just add the fields you need in the migration, and override the Media model
https://github.com/spatie/laravel-medialibrary/blob/293a066890e10c951ff29bc76bfa026a60034edc/config/media-library.php#L37
